From 2e372976d576f4497d338d2d3b0ecd64a4de5c11 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Mon, 8 Apr 2019 14:10:44 +0800
Subject: [PATCH 07/30] drm/imx/hdp: drop redundant drm_edid_to_eld() call in
 imx-hdp.c

According to upstream commit d471ed04b487 ("drm/drivers: drop
redundant drm_edid_to_eld() calls"), remove drm_edid_to_eld.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index 7a848fe3f625..13c4c0160eb1 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -887,9 +887,7 @@ static int imx_hdp_connector_get_modes(struct drm_connector *connector)
 				dev_warn(hdp->dev, "Invalid edid, use default video modes\n");
 				num_modes =
 					imx_hdp_default_video_modes(connector);
-			} else
-				/* Store the ELD */
-				drm_edid_to_eld(connector, edid);
+			}
 			kfree(edid);
 		} else {
 			dev_info(hdp->dev, "failed to get edid, use default video modes\n");
-- 
2.17.1

